rstudio/rhel9-python-3.9/Dockerfile

```diff
@@ -2,9 +2,10 @@ ARG BASE_IMAGE
 FROM ${BASE_IMAGE}
 
 # Access the client's secret for the subscription manager from the environment variable
-ARG SECRET_DIR=/opt/app-root/src/.sec
-ARG SERVERURL_DEFAULT=""
+RUN mkdir -p /opt/app-root/bin/Rpackages/4.3 && chmod -R a+w /opt/app-root/bin/Rpackages/4.3
```

Comment:
Please revert this change from here.

This block should replace this here: https://github.com/red-hat-data-services/notebooks/blob/main/rstudio/rhel9-python-3.9/Dockerfile#L54